_posts/2020-02-01-tuning-multiplexer/2020-02-01-tuning-multiplexer.Rmd

```diff
@@ -164,7 +163,7 @@ at = AutoTuner$new(
 )
 rr = resample(task, at, cv2, store_models = TRUE)
 # access 1st inner tuning result
-ll = rr$data$state[[1]]$model$tuning_instance
+ll = as.data.table(rr)$learner[[1]]$model$tuning_instance
```

Comment:
I am sure that there is a more elegant way to access the learner model.

Reply:
if you you need better ABs, please open issues / PRs!